src/lib/AnnotationControls.ts

```diff
@@ -11,7 +11,7 @@ export const CLASS_REGION_BUTTON = 'bp-AnnotationControls-regionBtn';
 export const CLASS_BUTTON_ACTIVE = 'is-active';
 export const CLASS_GROUP_HIDE = 'is-hidden';
 
-export enum AnnotationMode {
+export enum AnnotationType {
     HIGHLIGHT = 'highlight',
     NONE = 'none',
```

Comment:
If this is truly unrelated to Mode, this enum value should no longer be needed. The active control would just be undefined or null.